optimum/habana/transformers/training_args.py

```diff
@@ -178,6 +177,14 @@ class GaudiTrainingArguments(TrainingArguments):
         },
     )
 
+    half_precision_backend: str = field(
+        default="auto",
```

Comment:
```suggestion
        default="hpu_amp",
```
I would just set the default to `hpu_amp`. `auto` enables to automatically switch to `cuda_amp` or `cpu_amp` depending on the device that is being used. But in our case, `cpu_map` will be barely used.